velocity/tool/src/java/org/sakaiproject/util/ParameterParser.java

```diff
@@ -232,17 +232,17 @@ public FileItem getFileItem(String name)
 	{
 		// wrap the Apache FileItem in our own homegrown FileItem
 		Object o = m_req.getAttribute(name);
-		if (o != null && o instanceof org.apache.commons.fileupload.FileItem)
+		if (o instanceof org.apache.commons.fileupload.FileItem)
 		{
 			org.apache.commons.fileupload.FileItem item = (org.apache.commons.fileupload.FileItem) o;
-			try
-            {
-	            return new FileItem(Normalizer.normalize(item.getName(), Normalizer.Form.NFC), item.getContentType(), item.getInputStream());
-            }
-            catch (IOException e)
-            {
-            	return new FileItem(Normalizer.normalize(item.getName(), Normalizer.Form.NFC), item.getContentType(), item.get());
-            }
+			try {
+				return new FileItem(
+						Normalizer.normalize(item.getName(), Normalizer.Form.NFC),
+						item.getContentType(), item.getInputStream()
+				);
+			} catch (IOException e) {
+				// Ignore
```

Comment:
I don't like the pattern of logging in a utility class, I see it as the responsibility of the calling class to log if it's `null` and doesn't like that.

I think the better fix it to throw the IOException up to the caller, but I'm not sure it's worth fixing this one like that as using common-upload directly is the better way to go.